soen341-UB3-QA/routes/web.php

```diff
@@ -26,6 +26,11 @@
 
 Route::get('/question/{id}', 'QuestionController@show');
 Route::post('/reply/{id}', 'ReplyController@store');
+Route::get('/like/{id}', 'LikeController@like');
+Route::get('/dislike/{id}', 'LikeController@dislike');
+Route::get('/accept/{id}', 'LikeController@accept');
+Route::get('/reject/{id}', 'LikeController@reject');
+Route::get('/normalize/{id}', 'LikeController@normalize');
 
```

Comment:
Like, Dislike, Accept, Reject and Normalize should not all go to different base url.
It should go to /reply/1/{id} for like, It should go to /reply/2/{id} for dislike... etc.